cmd/collectors/zapi/plugins/shelf/shelf.go

```diff
@@ -55,81 +56,84 @@ func (my *Shelf) Init() error {
 		return err
 	}
 
-	my.query = "storage-shelf-environment-list-info"
+	if !my.client.IsClustered() {
 
```

Comment:
did you mean to add the rest of the method to this conditional block?

if so, better to
if my.client.IsClustered() {
return nil
}
and out-dent the block